tests/integration/test_api/test_config/test_max_upload_size/test_max_upload_size.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+from os.path import join, dirname, realpath
+from uuid import uuid4
+
+import pytest
+import requests
+
+from wazuh_testing.tools.configuration import check_apply_test, get_api_conf
+
+# Marks
+
+pytestmark = pytest.mark.server
+
+# Configurations
+
+test_data_path = join(dirname(realpath(__file__)), 'data')
+configurations_path = join(test_data_path, 'conf.yaml')
+configuration = get_api_conf(configurations_path)
+
+
+# Fixtures
+
+@pytest.fixture(scope='module', params=configuration)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+# Tests
+
+@pytest.mark.parametrize('tags_to_apply, expected_status_code', [
+    ({'limitless_upload_size'}, 200),
+    ({'low_upload_size'}, 413)
+])
```

Comment:
Configuration ids are not set properly, so the result is not very verbose:
```
get_configuration0-tags_to_apply0-200
```
Maybe something like this will do the work

```suggestion
configuration_ids=['limitless_upload_size','low_upload_size']
expected_responses = [200, 413]

# Fixtures

@pytest.fixture(scope='module', params=configuration, ids=configuration_ids)
def get_configuration(request):
    """Get configurations from the module."""
    return request.param


# Tests

@pytest.mark.parametrize('tags_to_apply, expected_status_code', [
    ({'limitless_upload_size'}, expected_responses[0]),
    ({'low_upload_size'}, expected_responses[1])
], ids=expected_responses)

```

Resulting in the following ids:
```
test_api/test_config/test_max_upload_size/test_max_upload_size.py::test_max_upload_size[limitless_upload_size-200]
test_api/test_config/test_max_upload_size/test_max_upload_size.py::test_max_upload_size[low_upload_size-413]

```